README.md

```diff
@@ -211,7 +212,8 @@ This code will output the following HTML:
 - codespan(*string* code)
 - br()
 - del(*string* text)
-- link(*string* href, *string* title, *string* text)
+- link(*string* href, *string* title, *string* text, [*boolean* checked]).
+  - `checked` only defined when `gfm` is `true` and there is a check box at the start of the list item (e.g. `* [ ] foo`).
```

Comment:
This file was moved to [USING_PRO.md](https://github.com/markedjs/marked/blob/master/docs/USING_PRO.md). and is causing merge conflicts.